modules/storage-persistent-storage-azure-file.adoc

```diff
@@ -44,12 +44,12 @@ spec:
     shareName: share-1 <4>
     readOnly: false
 ----
-<1> The name of the PersistentVolume.
-<2> The size of this PersistentVolume.
-<3> The name of the Secret that contains the Azure File share credentials.
+<1> The name of the persistent volume.
+<2> The size of this persistent volume.
+<3> The name of the `Secret` object that contains the Azure File share credentials.
```

Comment:
Or should we say, "The name of the secret that contains..." to be consistent with #1 and #2?